getting-started/logging-in.rst

```diff
@@ -0,0 +1,76 @@
+##########
+Logging in
+##########
+
+You need NCSA Kerberos credentials to log into the LSST Science Platform Notebook Aspect.
+Members of LSST receive an NCSA Kerberos account as part of your onboarding.
+If haven’t onboarded into LSST yet, talk to your T/CAM or supervisor (see `DM Onboarding <https://developer.lsst.io/team/onboarding.html>`__).
+
+Step 1: Use the NCSA VPN with Cisco AnyConnect
+==============================================
+
+As a development deployment, the Notebook Aspect is not yet directly accessible from the internet.
+For now, you’ll use NCSA’s VPN, through Cisco AnyConnect, to log into the Notebook Aspect.
+
+.. tip::
+
+   If you are on an approved network, like NOAO, you can skip the VPN and log in directly.
+
+To use the NCSA VPN, open `vpn.ncsa.illinois.edu <https://vpn.ncsa.illinois.edu>`__ in your web browser.
+Select ``ncsa-vpn-default`` from the **GROUP** menu and enter your NCSA Kerberos username and password.
+The page installs and starts the Cisco AnyConnect application for you.
+
+.. tip::
+
+   In the future, you can open the AnyConnect application and enter ``vpn.ncsa.illinois.edu`` to get connected.
+   There's no need to go back to the `vpn.ncsa.illinois.edu <https://vpn.ncsa.illinois.edu>`__ website.
+
+Step 2: Log in
+==============
+
+Open `lsst-lspdev.ncsa.illinois.edu/nb <https://lsst-lspdev.ncsa.illinois.edu/nb>`__ in your web browser. Click the **Sign in with CILogon** button, then enter your NCSA Kerberos credentials on the NCSA CILogon page.
+
+Once you click **Log in**, you’ll be redirected to the Notebook Aspect’s Spawner page.
```

Comment:
I have an extra authentication step, Duo, to have special access to the PDAC. Maybe better to say "After authentication is completed" instead of "Once you click Log In", to cover all scenarios.